hydra-node/src/Hydra/HeadLogic.hs

```diff
@@ -412,6 +415,8 @@ onOpenNetworkReqSn env ledger st otherParty sv sn requestedTxIds mDecommitTx =
   requireReqSn $
     -- Spec: wait ŝ = ̅S.s
     waitNoSnapshotInFlight $
+      -- Spec: wait v = v̂
+
```

Comment:
Must: not duplicate comments

Did you self-review the diff before asking for a review?

Reply:
There is something that duplicates these in my tool chain. I can't figure out why this happens and I think it does when I rebase sometimes. Need to figure it out.